spec/models/miq_provision_request_spec.rb

```diff
@@ -201,6 +201,12 @@ def create_request(user, vm_template, prov_options)
               {:count => 12, :memory => 8_589_938_688, :cpu => 32, :storage => 44.gigabytes}
             end
             it_behaves_like "check_quota"
+
+            it "invalid vm_template does not raise error" do
+              create_requests
+              MiqRequest.first.update_attributes(:vm_template => nil)
```

Comment:
The first request will now have no source (aliased to the template), which we have a presence validation on.